dev-util/bcc/bcc-0.22.0-r1.ebuild

```diff
@@ -72,6 +69,13 @@ src_prepare() {
 	bpf_link_path="$(realpath --relative-to="${S}/src/cc/libbpf" /usr/include/bpf)" || die
 	ln -sfn "${bpf_link_path}" src/cc/libbpf/include || die
 
+	# bug 811288
+	local script scriptname
+	for script in $(find tools/old -type f -name "*.py"); do
```

Comment:
Nits would be that we should have || die on the find (inside the subshell) and on basename too.

Reply:
Done